lifecycle/lifecycle-viewmodel-savedstate/src/commonMain/kotlin/androidx/lifecycle/AbstractSavedStateViewModelFactory.kt

```diff
@@ -53,12 +52,6 @@ public expect abstract class AbstractSavedStateViewModelFactory :
         defaultArgs: SavedState?
     )
 
-    @Deprecated(level = DeprecationLevel.HIDDEN, message = "Use constructor with SavedState")
```

Comment:
- This function seems wasn't deprecated in a stable release
- It is a public stable API (Bundle was available as `api` dependency)

Because of this, we need to provide backward compatibility, keeping this function

Reply:
@igordmn do you want to keep core-bundle in dependencies?

Reply:
Yes, as an artifact, not as a project